modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaCamelServerCodegen.java

```diff
@@ -171,7 +171,7 @@ public void addOperationToGroup(String tag, String resourcePath, Operation opera
     }
 
     private void addCliOptions() {
-        cliOptions.add(new CliOption(CAMEL_REST_COMPONENT, "name of the Camel component to use as the REST consumer").defaultValue(camelRestComponent));
+        cliOptions.add(new CliOption(CAMEL_REST_COMPONENT, "name of the Camel component to u as the REST consumer").defaultValue(camelRestComponent));
```

Comment:
unnecessary change

Reply:
I can't figure out where the change comes from